boot/zephyr/Kconfig

```diff
@@ -722,7 +722,7 @@ config MULTITHREADING
 	default y if BOOT_SERIAL_CDC_ACM #usb driver requires MULTITHREADING
 	default y if BOOT_USB_DFU_GPIO || BOOT_USB_DFU_WAIT
 	default n if SOC_FAMILY_NRF
-	default n if SOC_FAMILY_ESP32 && MCUBOOT
+	default n if SOC_FAMILY_ESPRESSIF_ESP32 && MCUBOOT
```

Comment:
@almir-okato, not related to this PR but, do we need `&& MCUBOOT` in there? This is mcuboot build, so `MCUBOOT` will always be set I think.

Reply:
I think you can remove the `&& MCUBOOT` part